charts/netbox-operator/ci/default-values.yaml

```diff
@@ -0,0 +1,4 @@
+host: demo.netbox.dev
+https: true
+auth:
+  apiToken: "55570df95c0c850ada67a4185228732aaffd4964"
```

Comment:
should there be a hardcoded token in here?

Reply:
@RangerRick No, but I'm waiting for an official instance provided by NetBox cloud 😇
This one is used for the demo instance, so it's all fine and probably deleted already.

Reply:
Fixed